go/libkb/globals.go

```diff
@@ -372,7 +372,10 @@ func (g *GlobalContext) ConfigureLogging(usage *Usage) error {
 	// Start redirecting logs if the logFile should be used
 	// If this is not called, prints logs to stdout.
 	if ok {
-		err := logger.SetLogFileConfig(g.Env.GetLogFileConfig(logFile))
+		err := logger.SetLogFileConfig(g.Env.GetLogFileConfig(logFile), &logger.BufferedLoggerConfig{
+			Frequency: time.Second,
+			Size:      1000000,
```

Comment:
A second is a lot, we may just entirely lose what caused a crash. Might be worth having a conversation in core about it. I'm surprised you don't get as-good results with like 200ms.

Reply:
Sure, I'll make it 200ms, the key change here is the buffer size anyway.

Reply:
If a second was arbitrary, can you test even smaller? we should go as small as we can without impacting performance.